static/docs/use-cases/data-registry.md

```diff
@@ -0,0 +1,163 @@
+# Data Registry
+
+We developed the `dvc get`, `dvc import`, and `dvc update` commands with the aim
+to enable reusability of any <abbr>data artifacts</abbr> (raw data, intermediate
+results, models, etc) between different projects. For example, project A may use
+a data file to begin its data [pipeline](/doc/command-reference/pipeline), but
+project B also requires this same file; Instead of
+[adding it](/doc/command-reference/add#example-single-file) it to both projects,
+B can simply import it from A.
+
+Taking this idea to a useful extreme, we could create a <abbr>project</abbr>
+that is exclusively dedicated to
+[tracking and versioning](/doc/use-cases/data-and-model-files-versioning)
+datasets (or any kind of large files) – by mainly using `dvc add` to build it.
+Such a project would not have [stages](/doc/command-reference/run), but its data
+files may be updated manually as they evolve. Other projects can then share
+these artifacts by downloading (`dvc get`) or importing (`dvc import`) them for
+use in different data processes – and these don't even have to be _DVC
+projects_, as `dvc get` works anywhere in your system.
+
+The advantages of using a data registry are:
+
+- Centralization: Data [shared](/doc/use-cases/share-data-and-model-files) by
+  multiple projects can be stored in a single location (with the ability to
+  create distributed copies on other remotes). This simplifies data management
+  and helps use storage space efficiently.
+- [Versioning](/doc/use-cases/data-and-model-files-versioning): Any version of
+  the stored data or ML modes can be used in other <abbr>projects</abbr> at any
+  time.
+- Persistence: The registry controlled
+  [remote storage](/doc/command-reference/remote) (e.g. an S3 bucket) improves
+  data security. There are less chances someone can delete or rewrite a model,
+  for example.
+- Lifecycle management: Manage your data like you do with code, leveraging Git
+  and GitHub features such as version history, pull requests, reviews, or even
+  continuous deployment of ML models.
+- Security: Registries can be setup to have read-only remote storage (e.g. an
+  HTTP location). Git versioning of DVC-files allows us to track and audit data
+  changes.
+- Reusability: Reproduce and organizing _feature stores_ with `dvc get` and
```

Comment:
reproduce and organize? I would also put this on top. Centralization (should we actually name in Storage Optimization for simplicity) - is good, but it's just an optimization, it's not what people usually care a lot initially :)

Reply:
> reproduce and organize?

I re-phrased your suggestion from https://github.com/iterative/dvc.org/pull/679#discussion_r337339419:  "Reusability - along with import/get this can serve as simple way to reproduce, or up to organizing feature stores."

> I would also put this on top.

Ok, moving under **Versioning**...

> Centralization...

Renaming **Storage Optimization**. I put it on top because it's the only benefit specific to data registries, not general to DVC projects, but OK moving it under *Persistence**...

Reply:
>  I put it on top because it's the only benefit specific to data registries, not general to DVC projects,

That not how I see at least. All these benefits of saving space are nice but secondary to major improvements to the workflow itself. I would describe the biggest benefit as _codification of data_. You don't have to rely on some random ad-hoc naming and location conventions when you version you data. You have you history. You manage updates with reviews. You have a simple API to access those artifacts with reusability (it's similar to the software packaging systems like pip). Those are major benefits to this. They are in the way you organize things in the first place.

> not general to DVC projects

this overlap does not matter as I see it. A regular DVC project is about managing complexity of a single project - reproducibility, history to compare results and being able to get back, etc, etc. Data registry - is a higher level use case. It's about organizing datasets and/or models in your team. If I get your point right, yes a regular DVC project exports data as well. And using a list of DVC projects can be totally another way of organizing a data registry. The only complexity would be to organize a mechanism to discover them.

Reply:
Ah woops I see what you meant originally. "organizing" -> "organize". Changing... Will also address your other comments here ^

Reply:
> I would describe the biggest benefit as _codification of data_...

Nice one! Added it under [Versioning] as "Data as code". The part about `You have you history. You manage updates with reviews.` is already under "Lifecycle management". The part about "reusability" is already under "Reusability" just below, but I added some keywords. (By "API" I assume you mean CLI.)

> Data registry - is a higher level use case...

Great point. I'm not super convinced but np: I understand this use case may serve as a landing page of someone reading about DVC for the first time, so its nice to have all the benefits here.

I just wonder if they should also be listed and even expanded on in other pages...

Reply:
> I understand this use case may serve as a landing page of someone reading about DVC for the first time, so its nice to have all the benefits here.

yep, exactly!!

Reply:
>By "API" I assume you mean CLI.

we have both actually

Reply:
Yes, but we're only using CLI so far in our docs and examples, so I changed the word to CLI. Plus I don't think sure the API has export/import functions yet.